.github/workflows/reusable-docs.yml

```diff
@@ -102,3 +102,38 @@ jobs:
     # Use "xvfb-run" since some doctest tests open GUI windows
     - name: 'Run documentation doctest'
       run: xvfb-run make -C Doc/ PYTHON=../python SPHINXERRORHANDLING="--fail-on-warning" doctest
+
+  epubcheck:
+    name: 'EPUB check'
+    runs-on: ubuntu-latest
+    timeout-minutes: 40
+    steps:
+    - name: 'Check out latest PR branch commit'
+      uses: actions/checkout@v4
+      with:
+        persist-credentials: false
+        ref: >-
+          ${{
+            github.event_name == 'pull_request'
+            && github.event.pull_request.head.sha
+            || ''
+          }}
+    - name: 'Set up Python'
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3'
+        cache: 'pip'
+        cache-dependency-path: 'Doc/requirements.txt'
+    - name: 'Install build dependencies'
+      run: make -C Doc/ venv
+    - name: 'Build and load EPUB checks to file'
+      continue-on-error: true
+      run: |
+        set -Eeuo pipefail
+        make -C Doc/ PYTHON=../python SPHINXOPTS="--quiet" epub
+        pip install epubcheck
+        epubcheck Doc/build/epub/Python.epub &> Doc/epubcheck.txt
+    - name: 'Check for FATAL errors in EPUB'
+      if: github.event_name == 'pull_request'
+      run: |
+        python Doc/tools/check-epub.py
```

Comment:
Rather than this new job, shall we add these two steps to the end of the existing `build-doc` job?

That can help prevent quite a bit of duplicate setup config.

Reply:
My intention here was to paralelize, but now I see that doctest is much longer than builds, so the length of the docs job shouldn't be a problem.